build/emoji/emoji_gen.mjs

```diff
@@ -375,8 +375,8 @@ const getSkinnedCategories = memoize(genSkinnedCategories);
 export const EmojiIndicesByCategory = new Map([${skinnedCats.join(', ')}]);
 `;
 
-// write emoji.jsx
-endResults.push(writeFile('emoji.jsx', 'utils/emoji.jsx', emojiJSX));
+// write emoji.tsx
+endResults.push(writeFile('emoji.tsx', 'utils/emoji.tsx', emojiJSX));
```

Comment:
Would you mind naming the file `emoji.ts` instead of `emoji.tsx`? I can't remember why we used a `.jsx` extension originally, but we don't need this file to have JSX support since it doesn't contain any React code

Reply:
I made the requested changes.